test/e2e/tests/e2e_test.go

```diff
@@ -122,6 +126,10 @@ func fetchBlockChain(ctx context.Context, t *testing.T) []*types.Block {
 
 	from := status.SyncInfo.EarliestBlockHeight
 	to := status.SyncInfo.LatestBlockHeight
+	// limit blocks to fetch to avoid long test times
+	if to-from > maxBlocks {
```

Comment:
Do you expect any issues if we sync less than `LatestBlockHeight`? Maybe print a warning here so you'll now that we didn't sync completely?

Reply:
No issues expected in e2e tests, we rarely go to > 100 blocks.

It does make sense on local testing, using long-running e2e network (but this is really rarely used, obly when debugging)